easybuild/easyconfigs/b/Bison/Bison-2.7-ClangGCC-1.7.20.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'Bison'
+version = '2.7'
```

Comment:
this Bison easyconfig is needed as a dep for the MVAPICH2 easyconfig, the existing MVAPICH2 easyconfig for the same version already uses Bison 2.7, so it makes sense to stick to this version

the only real solution for this version madness is supporting open-ended version specs (which is a whole new league of pain by itself, imho)